### PR TITLE
Create Footer

### PR DESCRIPTION
**Pull request by benclmnt**

### CODE REVIEW

File: functions/submission-created.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    console.log(`Received contact form submission from ${email}`);
+
+    // set up email msg. We are sending from our self to ourself.
+    const responseSubject = encodeURIComponent(
+        `Regarding your Angklung Ensemble enquiry, ${name}`
+    );
+    const msg = {
+        to: ANGKLUNG_EMAIL,
+        from: ANGKLUNG_EMAIL,
```
Comment (Yehezkiel01) [inline functions/submission-created.js:20]: Just to confirm, is this intentional to send from and to "ANGKLUNG_EMAIL"?
If that's the case, could you add a little comment there? Just to clarify doubts like the one that I have.

File: functions/submission-created.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+sgMail.setApiKey(SENDGRID_API_KEY);
+
+exports.handler = async function (event, context, callback) {
+    const { name, email, message, formType } = JSON.parse(
+        event.body
+    ).payload.data;
+
+    console.log(`Received contact form submission from ${email}`);
+
+    // set up email msg. We are sending from our self to ourself.
```
Comment (Yehezkiel01) [inline functions/submission-created.js:14]:
```suggestion
    // set up email msg. We are sending from ourself to ourself.
```

Comment (benclmnt) [reply functions/submission-created.js:14]: Probably will need to change the `to` email to "hello@\<our domain\>", after we buy a domain

**Review by Yehezkiel01 — APPROVED** [review]
LGTM! Thanks for the hardwork :D

**Pull request merged**